README.md

```diff
@@ -3,23 +3,43 @@ Train robotics model with integrated curriculum learning-based gripper environme
 
 Master's thesis [PDF](https://github.com/BarisYazici/masters_thesis/blob/master/final_report.pdf)
 
-### Prerequisites
+### Prerequisites (CPU)
 
 Install anaconda. Start a clean conda environment.
 
 ```
 conda create -n grasp_env python=3.6
 conda activate grasp_env
 ```
+python manipulation_main/training/train_stable_baselines.py train --config config/gripper_grasp.yaml --algo SAC --model_dir trained_models/SAC_full --timestep 100000 -v
+### Prerequisites (GPU)
+```
+conda create -n grasp_env python=3.6
+conda activate grasp_env
+conda install -c conda-forge cudatoolkit=10.0 cudnn=7.6.5
```

Comment:
Is conda-forge also necessary to install?

Reply:
Yes, conda-forge is necesary for CUDA build.
I found this: https://conda-forge.org/blog/posts/2021-11-03-tensorflow-gpu/
I added at the beginning because that was on here: https://www.tensorflow.org/install/pip